src/LiteCore/src/LiteCore.Shared/Interop/C4Database_native.cs

```diff
@@ -117,10 +117,6 @@ public static string c4db_getPath(C4Database* db)
         [return: MarshalAs(UnmanagedType.U1)]
         public static extern bool c4db_endTransaction(C4Database* database, [MarshalAs(UnmanagedType.U1)]bool commit, C4Error* outError);
 
-        [DllImport(Constants.DllName, CallingConvention = CallingConvention.Cdecl)]
-        [return: MarshalAs(UnmanagedType.U1)]
-        public static extern bool c4db_isInTransaction(C4Database* database);
```

Comment:
What does this have to do with the PR?

Reply:
Somehow this PR carries over the commit that I made but not yet pushed to the master branch when I created this PR branch from master branch... The reason I didn't push it is because I don't want a new build just for those changes... Let me take them out and create another PR for it.